packages/tradershub/src/features/cfd/components/MT5PlatformsList/MT5PlatformsList.tsx

```diff
@@ -48,14 +49,20 @@ const MT5PlatformsList = ({ onMT5PlatformListLoaded }: TMT5PlatformsListProps) =
                 </div>
             )}
             {isFetchedAfterMount &&
-                data?.map(account => {
-                    if (account.is_added)
-                        return <AddedMT5AccountsList account={account} key={`added-mt5-list-${account.loginid}`} />;
+                data?.map(MT5Account => {
+                    if (
+                        MT5Account.is_added &&
+                        MT5Account.is_virtual === activeTradingAccount?.is_virtual &&
+                        MT5Account.account_type === accountType
```

Comment:
[Question]: Another question, why do we need to check for the account type?

Reply:
We need to match the MT5 account type with the `accountType` this  `accountType` refer to the accountType switcher which I also think need a better name to make it clear. 


<img width="112" alt="Screenshot 2024-02-19 at 15 07 47" src="https://github.com/binary-com/deriv-app/assets/64970259/baa64dfc-cdf4-4c69-87bd-65cac895bf78">